app/backend/src/couchers/servicers/auth.py

```diff
@@ -354,11 +354,18 @@ def Deauthenticate(self, request, context):
         token = parse_session_cookie(dict(context.invocation_metadata()))
         logger.info(f"Deauthenticate(token={token})")
 
-        if token and self._delete_session(token):
-            return empty_pb2.Empty()
-        else:
-            # probably caused by token not existing
-            context.abort(grpc.StatusCode.UNKNOWN, errors.LOGOUT_FAILED)
+        # if we had a token, try to remove the session
+        if token:
+            self._delete_session(token)
+
+        # set the cookie to an empty string and expire immediately, should remove it from the browser
+        context.send_initial_metadata(
+            [
+                ("set-cookie", create_session_cookie("", now())),
```

Comment:
Would be easier if we used Max-Age insead of Expires, but that's a separate change.